Source/Core/DolphinQt/QtUtils/RightClickButton.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <QMouseEvent>
```

Comment:
This include can be a forward declaration

Reply:
Okay, I have done that. 👍 Can you explain why forward declarations are preferred over includes? Is it just a way to reduce build times?

Reply:
Yes and it also avoids pulling in identifiers that may be included inside of another header, reducing the amount of things put into scope (this also makes it easier to prevent code from unintentionally depending on the includes of another header).